tests/test.rs

```diff
@@ -196,10 +196,41 @@ fn test_from_char_iterator() {
         ("사회과학원 어학연구소", true),
         // String containing diverse characters
         ("表ポあA鷗ŒéＢ逍Üßªąñ丂㐀𠀀", true),
+        // String which has too many characters to even consider inlining
+        ("☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺☺", true),
```

Comment:
If I understand it correctly this wouldn't be inlined even if it was small enough due to containing non-ascii characters. Could you turn this into a long string of a's?